site/content/en/docs/Guides/feature-stages.md

```diff
@@ -38,6 +38,9 @@ The current set of `alpha` and `beta` feature gates:
 | [Multiple dynamic port ranges](https://github.com/googleforgames/agones/issues/1911)                                        | `PortRanges`                       | Disabled | `Alpha` | 1.41.0 |
 | [Port Policy None](https://github.com/googleforgames/agones/issues/3804)                                                    | `PortPolicyNone`                   | Disabled | `Alpha` | 1.41.0 |
 | [Scheduled Fleet Autoscaling](https://github.com/googleforgames/agones/issues/3008)                                         | `ScheduledAutoscaler`              | Disabled | `Alpha` | 1.43.0 |
+{{% feature publishVersion="1.48.0" %}}
```

Comment:
Mid table items tend to render badly, you may need to copy the whole table. But I'll run it and we can see what it looks like in preview.

(`make site-server` will give you a local version if you would like to try it).

Reply:
![image](https://github.com/user-attachments/assets/d1bd41b3-d3ad-40fe-bac7-19b40289113e)

https://1d43bbe-dot-preview-dot-agones-images.appspot.com/site/docs/guides/feature-stages/#feature-gates

Going to have to copy the whole block, but otherwise, I think this is good to go.